hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -118,7 +120,7 @@ public HoodieTableMetaClient(Configuration conf, String basePath, boolean loadAc
                                ConsistencyGuardConfig consistencyGuardConfig, Option<TimelineLayoutVersion> layoutVersion,
                                String payloadClassName) {
     LOG.info("Loading HoodieTableMetaClient from " + basePath);
-    this.basePath = basePath;
+    this.basePath = FilenameUtils.separatorsToUnix(basePath);
```

Comment:
This change seems unnecessary， WDYT?

Reply:
If you do not add this line of code, the org.apache.hudi.common.table.TestHoodieTableMetaClient#checkMetadata  test will not pass. because of org.apache.hudi.common.testutils.HoodieCommonTestHarness#tempDir using java.nio.file.Path, windows users cannot pass the test. but now the ci seems like failed.

Reply:
> If you do not add this line of code, the org.apache.hudi.common.table.TestHoodieTableMetaClient#checkMetadata test will not pass. because of org.apache.hudi.common.testutils.HoodieCommonTestHarness#tempDir using java.nio.file.Path, windows users cannot pass the test. but now the ci seems like failed.

ok， then

Reply:
The ci failed, please fix it
```
[INFO] There are 2 errors reported by Checkstyle 8.18 with style/checkstyle.xml ruleset.
[ERROR] src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java:[48,1] (imports) IllegalImport: Illegal import - org.apache.commons.io.FilenameUtils.
[ERROR] src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java:[50,8] (imports) UnusedImports: Unused import - java.io.File.
```

Reply:
fixed  and then ？